pkgs/development/python-modules/h3/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, buildPythonPackage
+, cmake
+, fetchPypi
+, pkgs
```

Comment:
The reason why I'm passing `pkg` is because I'm defining the python package `h3` here, while another package (outside package) is also called h3. When I put in `h3` here, it goes into an infinite loop. Not sure how to access top-level packages without accessing it through pkgs, unless you pass it to `callPackage` args maybe?